raven/breadcrumbs.py

```diff
@@ -28,40 +32,59 @@ def event_payload_considered_equal(a, b):
 
 class BreadcrumbBuffer(object):
 
-    def __init__(self, limit=100):
+    def __init__(self, limit=100, message_max_length=1024):
```

Comment:
I'll set `limit` and `message_max_length` as settings in the new-settings-branch